### PR TITLE
Filipino Translation - 1113 translated words

### PR DESCRIPTION
**Pull request by mind-blast**
recording-changes.asc - 618 translated words
1-git-tools.asc - 174 translated words
advanced-merging.asc - 321 translated words

Code text and computer code text are not counted as translated words.

### CODE REVIEW

File: book/02-git-basics/sections/recording-changes.asc [hunk 1]
```diff
@@ -465,12 +465,12 @@ Tandaan na ang commit ay nagtatala ng snapshot na iyong nai-setup sa iyong itina
...
 Sa bawat oras na magsagawa ka ng commit, Nagtatala ka ng isang snapshot sa iyong proyekto na pwede mong ibalik sa o ihambing sa ibang pagkakataon.
 
-==== Skipping the Staging Area
+==== Paglaktaw sa Staging na Lawak
 
 (((staging area, skipping)))
-Although it can be amazingly useful for crafting commits exactly how you want them, the staging area is sometimes a bit more complex than you need in your workflow.
-If you want to skip the staging area, Git provides a simple shortcut.
-Adding the `-a` option to the `git commit` command makes Git automatically stage every file that is already tracked before doing the commit, letting you skip the `git add` part:
+Kahit na ito ay maaaring kamangha-manghang kapki-pakinabang para sa pag-craft ng mga commit na eksakto sa kung ano ang gusto mo sa kanila, ang staging na lawak ay paminsan-minsan isang kaunting kumplikado kaysa sa kailangan mo sa iyong workflow.
```
Comment (josteinfrancois) [inline book/02-git-basics/sections/recording-changes.asc]: - update `kamangha-manghang kapki-pakinaban` to `nakamamanghang kapaki-pakinabang`
- you could also change `pag-craft` to `paglikha`

File: book/02-git-basics/sections/recording-changes.asc [hunk 2]
```diff
@@ -465,12 +465,12 @@ Tandaan na ang commit ay nagtatala ng snapshot na iyong nai-setup sa iyong itina
...
+Kung gusto mong laktawan ang staging na lawak, ang Git ay nagbibigay na isang simpleng shortcut.
```
Comment (josteinfrancois) [inline book/02-git-basics/sections/recording-changes.asc]: you could change `shortcut` to `daang tuwiran`

File: book/02-git-basics/sections/recording-changes.asc [hunk 3]
```diff
@@ -489,18 +489,18 @@ $ git commit -a -m 'added new benchmarks'
...
+==== Pag-alis ng mga File
 
 (((files, removing)))
-To remove a file from Git, you have to remove it from your tracked files (more accurately, remove it from your staging area) and then commit.
-The `git rm` command does that, and also removes the file from your working directory so you don't see it as an untracked file the next time around.
+Para mag-alis ng isang file mula sa Git, kailangan mong alisin ito mula sa iyong nabubaybayan na mga file (mas sakto, tanggalin ito mula sa iyong staging na lawak) at pagkatapos i-commit.
+Ang `git rm` na utos ay ginagawa yan, at inaalis din ang file mula sa iyong tinatrabahong direktoryo para hindi mo makikita ito bilang hindi nasubaybayan na file sa susunod.
 
-If you simply remove the file from your working directory, it shows up under the ``Changes not staged for commit'' (that is, _unstaged_) area of your `git status` output:
+Kung alisin mo lamang ang file mula sa iyong tinatrabahong direktoryo, If you simply remove the file from your working directory, Nagpapakita ito sa ilalim ng ``Ang mga pagbabago ay hindi pa na-staged para i-commit'' (yan ay, _unstaged_) lawak ng iyong `git status` na output:
```
Comment (josteinfrancois) [inline book/02-git-basics/sections/recording-changes.asc]: change `na-staged` to `na-stage`

File: book/02-git-basics/sections/recording-changes.asc [hunk 4]
```diff
@@ -517,7 +517,7 @@ Changes not staged for commit:
 no changes added to commit (use "git add" and/or "git commit -a")
 ----
 
-Then, if you run `git rm`, it stages the file's removal:
+Pagkatapos, kung ikaw ay nagpatakbo ng `git rm`, ito ay mag-stage sa pag-alis ng file:
```
Comment (josteinfrancois) [inline book/02-git-basics/sections/recording-changes.asc]: change `mag-stage` to `magsi-stage`

File: book/02-git-basics/sections/recording-changes.asc [hunk 5]
```diff
@@ -532,58 +532,58 @@ Changes to be committed:
     deleted:    PROJECTS.md
 ----
 
-The next time you commit, the file will be gone and no longer tracked.
-If you modified the file and added it to the staging area already, you must force the removal with the `-f` option.
-This is a safety feature to prevent accidental removal of data that hasn't yet been recorded in a snapshot and that can't be recovered from Git.
+Sa susunod na ikaw ay mag-commit, ang file ay mawawala at hindi na masubaybayan.
+Kung ikaw ay makapagbago ng isang file at madagdag ito sa stanging na lawak na, dapat mong pilitin ang pagtanggal kasama ang `-f` na opsyon.
```
Comment (josteinfrancois) [inline book/02-git-basics/sections/recording-changes.asc]: change `Kung ikaw ay makapagbago ng isang file at madagdag ito sa stanging na lawak na` to `Kung ikaw ay nakapagbago ng isang file at idinagdag na ito sa stanging na lawak`

File: book/02-git-basics/sections/recording-changes.asc [hunk 6]
```diff
@@ -532,58 +532,58 @@ Changes to be committed:
...
-==== Moving Files
+==== Paggalaw ng mga File
 
 (((files, moving)))
-Unlike many other VCS systems, Git doesn't explicitly track file movement.
-If you rename a file in Git, no metadata is stored in Git that tells it you renamed the file.
-However, Git is pretty smart about figuring that out after the fact -- we'll deal with detecting file movement a bit later.
+Hindi tulad ng maraming iba pang mga sistema ng VCS, ang Git ay hindi malinaw na nagsubaybay ng pagkilos ng file.
+Kung ikaw ay nagpalit ng pangalan sa Git, walang metadata ay naka-imbak sa Git na nagsasabi na iyong pinalitan ang pangalan ng file.
+Gayunpaman, Ang Git ay medyo matalino sa pag-uunawa na pagkatapos ng katotohanan -- haharapin natin ang pag-detect sa paggalaw ng file na mamayang kaunti.
```
Comment (josteinfrancois) [inline book/02-git-basics/sections/recording-changes.asc]: `Ang` to `ang`

Comment (josteinfrancois) [inline book/02-git-basics/sections/recording-changes.asc]: change `na mamayang kaunti` to `sa isang saglit`

File: book/02-git-basics/sections/recording-changes.asc [hunk 7]
```diff
@@ -606,6 +606,6 @@ $ git rm README.md
 $ git add README
 ----
 
-Git figures out that it's a rename implicitly, so it doesn't matter if you rename a file that way or with the `mv` command.
-The only real difference is that `git mv` is one command instead of three -- it's a convenience function.
-More importantly, you can use any tool you like to rename a file, and address the add/rm later, before you commit.
+Ang Git ay nakatukoy na ito ay isang pagpapahiwatig ng pagpalit ng pangalan, kaya hindi mahalaga na kung ikaw ay nakapalit ng pangalan sa file sa ganun na paraan o kasama ang `mv` na utos.
+Ang tanging tunay na kaibahan ay ang `git mv` ay isang utos sa halip na tatlo -- ito ay isang kaginhawahang function.
```
Comment (josteinfrancois) [inline book/02-git-basics/sections/recording-changes.asc]: `kaginhawahang ` to `maginhawang`

File: book/02-git-basics/sections/recording-changes.asc [hunk 8]
```diff
@@ -606,6 +606,6 @@ $ git rm README.md
...
+Higit na mahalaga, maaari kang gumamit ng anumang kasangkapan na gusto mong ipalit na pangalan sa isang file, at maipasa sa add/rm mamaya, bago ka mag-commit.
```
Comment (josteinfrancois) [inline book/02-git-basics/sections/recording-changes.asc]: `gusto mong ipalit na pangalan` to `gusto mong magpalit ng pangalan`

File: book/07-git-tools/1-git-tools.asc [hunk 9]
```diff
@@ -1,10 +1,10 @@
 [[_git_tools]]
-== Git Tools
+== Mga Git na Kasangkapan
 
-By now, you’ve learned most of the day-to-day commands and workflows that you need to manage or maintain a Git repository for your source code control.
-You’ve accomplished the basic tasks of tracking and committing files, and you’ve harnessed the power of the staging area and lightweight topic branching and merging.
+Sa ngayon, natutunan mo ang karamihan ng araw-araw sa mga utos at mga workflow na kailangan mong pamahalaan o pangalagaan ang Git na repositoryo para sa iyong source code na kontrol.
```
Comment (josteinfrancois) [inline book/07-git-tools/1-git-tools.asc]: `araw-araw` to `pang-araw-araw`

File: book/07-git-tools/1-git-tools.asc [hunk 10]
```diff
@@ -1,10 +1,10 @@
...
+Nagawa mo na ang mga pangunahing gawain ng pagsubaybay at pag-commit ng mga file, at na-harness mo ang kapayarihan sa stanging na lawak at magaan na paksa na pag-branch at pag-merge.
```
Comment (josteinfrancois) [inline book/07-git-tools/1-git-tools.asc]: - translate `na-harness` to `napakinabangan`
- `stanging` to `staging`

File: book/07-git-tools/1-git-tools.asc [hunk 11]
```diff
@@ -1,10 +1,10 @@
...
 
-Now you’ll explore a number of very powerful things that Git can do that you may not necessarily use on a day-to-day basis but that you may need at some point.
+Ngayon ikaw ay magsiyasat ng bilang sa pinaka kapangyarihan na bagay na magagawa ng Git na maaaring hindi mo na kinakailangang gamitin ng araw-araw ngunit maaaring kailangan mo sa isang punto.
```
Comment (josteinfrancois) [inline book/07-git-tools/1-git-tools.asc]: - `pinaka kapangyarihan` to `sobrang makapangyarihan`
- `ng araw-araw` to `sa pang-araw-araw`

File: book/07-git-tools/1-git-tools.asc [hunk 12]
```diff
@@ -34,9 +34,9 @@ include::sections/replace.asc[]
...
 include::sections/credentials.asc[]
 
-=== Summary
+=== Buod
 
-You’ve seen a number of advanced tools that allow you to manipulate your commits and staging area more precisely.
-When you notice issues, you should be able to easily figure out what commit introduced them, when, and by whom.
-If you want to use subprojects in your project, you’ve learned how to accommodate those needs.
-At this point, you should be able to do most of the things in Git that you’ll need on the command line day to day and feel comfortable doing so.
+Ikaw ay makakita ng bilang ng advanced na mga kasangkapan na nagpapahintulot sa iyo na manipulahin ang iyong mga commit at stanging na lawak na mas tumpak.
```
Comment (josteinfrancois) [inline book/07-git-tools/1-git-tools.asc]: `stanging` to `staging`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 13]
```diff
@@ -1,16 +1,16 @@
 [[_advanced_merging]]
-=== Advanced Merging
+=== Advanced na Merging
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: `Merging` to `Pag-merge`

File: book/07-git-tools/sections/advanced-merging.asc [hunk 14]
```diff
@@ -1,16 +1,16 @@
...
 
-Merging in Git is typically fairly easy.
-Since Git makes it easy to merge another branch multiple times, it means that you can have a very long lived branch but you can keep it up to date as you go, solving small conflicts often, rather than be surprised by one enormous conflict at the end of the series.
+Ang merging sa Git ay karaniwang medyo madali.
```
Comment (josteinfrancois) [inline book/07-git-tools/sections/advanced-merging.asc]: `merging` to `pag-merge`

**Review by josteinfrancois — COMMENTED** [review]
Hello @robin-ho. I have added several suggestions on your translations. Please check them all out then you're good to go.

File: book/02-git-basics/sections/recording-changes.asc [hunk 15]
```diff
@@ -465,12 +465,12 @@ Tandaan na ang commit ay nagtatala ng snapshot na iyong nai-setup sa iyong itina
 Anuman ang hindi mo naitanghal ay nakaupo pa rin doon na nabago; maaari mong gawin ang isa pang commit para madagdag sa iyong kasaysayan.
 Sa bawat oras na magsagawa ka ng commit, Nagtatala ka ng isang snapshot sa iyong proyekto na pwede mong ibalik sa o ihambing sa ibang pagkakataon.
 
-==== Skipping the Staging Area
+==== Paglaktaw sa Staging na Lawak
```
Comment (heywens) [inline book/02-git-basics/sections/recording-changes.asc]: retain ``Staging Area``

File: book/02-git-basics/sections/recording-changes.asc [hunk 16]
```diff
@@ -489,18 +489,18 @@ $ git commit -a -m 'added new benchmarks'
  1 file changed, 5 insertions(+), 0 deletions(-)
 ----
 
-Notice how you don't have to run `git add` on the `CONTRIBUTING.md` file in this case before you commit.
-That's because the `-a` flag includes all changed files.
-This is convenient, but be careful; sometimes this flag will cause you to include unwanted changes.
+Pansinin kung paano hindi ka kailangan na magpatakbo ng `git add` sa `CONTRIBUTING.md` na file sa kasong ito bago ikaw mag-commit.
```
Comment (heywens) [inline book/02-git-basics/sections/recording-changes.asc]: > bago ikaw mag-commit
to
> bago ka mag-commit

File: book/02-git-basics/sections/recording-changes.asc [hunk 17]
```diff
@@ -489,18 +489,18 @@ $ git commit -a -m 'added new benchmarks'
...
+Iyan ay dahil ang `-a` na flag ay nagsasama ng lahat na nabago na mga file.
+Ito ay maginhawa, ngunit magingat; kung minsan ang flag na ito ay magdudulot sa iyo ng pagsasama ng hindi inaasahang mga pagbabago.
```
Comment (heywens) [inline book/02-git-basics/sections/recording-changes.asc]: ``magingat`` to ``mag-ingat``

File: book/02-git-basics/sections/recording-changes.asc [hunk 18]
```diff
@@ -489,18 +489,18 @@ $ git commit -a -m 'added new benchmarks'
...
 
 [[_removing_files]]
-==== Removing Files
+==== Pag-alis ng mga File
 
 (((files, removing)))
-To remove a file from Git, you have to remove it from your tracked files (more accurately, remove it from your staging area) and then commit.
-The `git rm` command does that, and also removes the file from your working directory so you don't see it as an untracked file the next time around.
+Para mag-alis ng isang file mula sa Git, kailangan mong alisin ito mula sa iyong nabubaybayan na mga file (mas sakto, tanggalin ito mula sa iyong staging na lawak) at pagkatapos i-commit.
```
Comment (heywens) [inline book/02-git-basics/sections/recording-changes.asc]: * ``nabubaybayan`` to ``nasubaybayan``
* you can also change ``sakto`` to ``tama``

File: book/02-git-basics/sections/recording-changes.asc (hunk 3 above)
Comment (heywens) [inline book/02-git-basics/sections/recording-changes.asc]: * source phrase not removed. kindly remove this:
> If you simply remove the file from your working directory

Comment (heywens) [inline book/02-git-basics/sections/recording-changes.asc]: * ``Nagpapakita`` to lower case > ``nagpapakita``
> Ang mga pagbabago ay hindi pa na-staged > remove pa (yet)

File: book/02-git-basics/sections/recording-changes.asc (hunk 5 above)
Comment (heywens) [inline book/02-git-basics/sections/recording-changes.asc]: ``stanging`` to ``staging``

File: book/02-git-basics/sections/recording-changes.asc [hunk 19]
```diff
@@ -532,58 +532,58 @@ Changes to be committed:
...
-This is a safety feature to prevent accidental removal of data that hasn't yet been recorded in a snapshot and that can't be recovered from Git.
+Sa susunod na ikaw ay mag-commit, ang file ay mawawala at hindi na masubaybayan.
+Kung ikaw ay makapagbago ng isang file at madagdag ito sa stanging na lawak na, dapat mong pilitin ang pagtanggal kasama ang `-f` na opsyon.
+Ito ay isang tampok na kaligtasan para maiwasan ang aksidenteng pag-alis ng datos na hindi pa naitala sa isang snapshot at hindi na ito makuha muli mula sa Git.
 
-Another useful thing you may want to do is to keep the file in your working tree but remove it from your staging area.
-In other words, you may want to keep the file on your hard drive but not have Git track it anymore.
-This is particularly useful if you forgot to add something to your `.gitignore` file and accidentally staged it, like a large log file or a bunch of `.a` compiled files.
-To do this, use the `--cached` option:
+Isa pang kapaki-pakinabang na bagay na gusto mong gawin ay ang pagpanatili ng file sa iyong tinatrabahong tree ngunit alisin ito mula sa iyong stanging na lawak.
```
Comment (heywens) [inline book/02-git-basics/sections/recording-changes.asc]: ``stanging`` to ``staging``

File: book/02-git-basics/sections/recording-changes.asc [hunk 20]
```diff
@@ -532,58 +532,58 @@ Changes to be committed:
...
+Kung gusto mong palitan ng pangalan ang isang file sa Git, maaari kang magpatakbo ng isang bagay na tulad nito:
 
 [source,console]
 ----
 $ git mv file_from file_to
 ----
 
-and it works fine.
-In fact, if you run something like this and look at the status, you'll see that Git considers it a renamed file:
+at ito ay gumagana ng maayos.
```
Comment (heywens) [inline book/02-git-basics/sections/recording-changes.asc]: ``ng`` to ``nang`` (adverb)

File: book/02-git-basics/sections/recording-changes.asc [hunk 21]
```diff
@@ -532,58 +532,58 @@ Changes to be committed:
...
+Sa katunayan, kung ikaw ay magpatakbo ng isang bagay na tulad nito at tumigin sa katayuan, iyong makikita na ang Git ay isinaalang-alang ang pagbago ng panglan ng file:
```
Comment (heywens) [inline book/02-git-basics/sections/recording-changes.asc]: ``tumigin`` to ``tumingin``

File: book/02-git-basics/sections/recording-changes.asc (hunk 7 above)
Comment (heywens) [reply book/02-git-basics/sections/recording-changes.asc]: ``kaginhawahang`` to ``kaginhawaang``

File: book/02-git-basics/sections/recording-changes.asc (hunk 8 above)
Comment (heywens) [inline book/02-git-basics/sections/recording-changes.asc]: or maaari kang gumamit ng anumang kasangkapan na gusto mo upang magpalit na pangalan sa isang file

File: book/07-git-tools/1-git-tools.asc (hunk 9 above)
Comment (heywens) [reply book/07-git-tools/1-git-tools.asc]: * pang-araw-araw na mga utos
* para sa iyong source code na kontrol
to ``para sa kontrol ng iyong source code``

File: book/07-git-tools/sections/advanced-merging.asc [hunk 22]
```diff
@@ -1,16 +1,16 @@
...
-Therefore, if you wait too long to merge two branches that diverge quickly, you can run into some issues.
+Gayunpaman, minsan nakakalito ang mga kasalungat na magaganap.
+Di tulad ng ibang mga bersyon na kontrol na mga sistema, ang Git ay hindi susubok upang maging sobrang matalino tungkol sa pagsama ng pagsasalungatan na resolusyon.
+Ang pilisopiya ng Git ay dapat maging talino tungkol sa pagtukoy kapag ang isang merge na resolusyon ay hindi malabo, ngunit kung may isang salungat, hindi ito sinubukan na maging matalino tungkol sa awtomatikong palutas nito.
+Samakatuwid, kung ikaw ay maghintay ng sobrang matagal upang i-merge ang dalawang branch na mabilis na nagbabagu-bago, maaari kang magpatakbo ng ilang mga isyu.
 
-In this section, we'll go over what some of those issues might be and what tools Git gives you to help handle these more tricky situations.
-We'll also cover some of the different, non-standard types of merges you can do, as well as see how to back out of merges that you've done.
+Sa seksyon na ito, kami ay magpapatuloy kung ano ang mga maaring isyu at anong mga kasangkapan na nagbibigay tulong upang panghawakan ang mga ito na mas madayang mga sitwasyon.
```
Comment (heywens) [inline book/07-git-tools/sections/advanced-merging.asc]: ``maaring`` to ``maaaring``

File: book/07-git-tools/sections/advanced-merging.asc [hunk 23]
```diff
@@ -1,16 +1,16 @@
...
 
-However, sometimes tricky conflicts do occur.
-Unlike some other version control systems, Git does not try to be overly clever about merge conflict resolution.
-Git's philosophy is to be smart about determining when a merge resolution is unambiguous, but if there is a conflict, it does not try to be clever about automatically resolving it.
-Therefore, if you wait too long to merge two branches that diverge quickly, you can run into some issues.
+Gayunpaman, minsan nakakalito ang mga kasalungat na magaganap.
+Di tulad ng ibang mga bersyon na kontrol na mga sistema, ang Git ay hindi susubok upang maging sobrang matalino tungkol sa pagsama ng pagsasalungatan na resolusyon.
+Ang pilisopiya ng Git ay dapat maging talino tungkol sa pagtukoy kapag ang isang merge na resolusyon ay hindi malabo, ngunit kung may isang salungat, hindi ito sinubukan na maging matalino tungkol sa awtomatikong palutas nito.
+Samakatuwid, kung ikaw ay maghintay ng sobrang matagal upang i-merge ang dalawang branch na mabilis na nagbabagu-bago, maaari kang magpatakbo ng ilang mga isyu.
```
Comment (heywens) [inline book/07-git-tools/sections/advanced-merging.asc]: ``maghintay ng sobrang matagal``
to
``naghintay nang sobrang tagal``

File: book/07-git-tools/sections/advanced-merging.asc [hunk 24]
```diff
@@ -1,16 +1,16 @@
...
-Since Git makes it easy to merge another branch multiple times, it means that you can have a very long lived branch but you can keep it up to date as you go, solving small conflicts often, rather than be surprised by one enormous conflict at the end of the series.
+Ang merging sa Git ay karaniwang medyo madali.
+Dahil ang Git ay nagagawa nito ipadali ang pag-merge sa ibang branch ng maraming beses, ito ay nangangahulugan na maaari ka na merong isang matagal nang buhay na branch ngunit maaari mong panatilihin ang pinakabago habang ikaw ay gumagamit, paglutas ng maliit na mga kasalungat ng madalas, sa halip na mabigla ng isang napakalaking salungat sa katapusan ng serye.
 
-However, sometimes tricky conflicts do occur.
-Unlike some other version control systems, Git does not try to be overly clever about merge conflict resolution.
-Git's philosophy is to be smart about determining when a merge resolution is unambiguous, but if there is a conflict, it does not try to be clever about automatically resolving it.
-Therefore, if you wait too long to merge two branches that diverge quickly, you can run into some issues.
+Gayunpaman, minsan nakakalito ang mga kasalungat na magaganap.
```
Comment (heywens) [inline book/07-git-tools/sections/advanced-merging.asc]: ``kasalungat`` (adjective) to ``salungatan`` (noun)

**Review by heywens — COMMENTED** [review]
Hi Sir @robin-ho,

I've added few suggestions in your translation.
Kindly check.

Thanks.

File: book/02-git-basics/sections/recording-changes.asc (hunk 1 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 2 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 3 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 4 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 5 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 6 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 7 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change following kaginhawaang

File: book/02-git-basics/sections/recording-changes.asc (hunk 8 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/07-git-tools/1-git-tools.asc (hunk 9 above)
Comment (mind-blast) [reply book/07-git-tools/1-git-tools.asc]: done change

File: book/07-git-tools/1-git-tools.asc (hunk 10 above)
Comment (mind-blast) [reply book/07-git-tools/1-git-tools.asc]: done change

File: book/07-git-tools/1-git-tools.asc (hunk 11 above)
Comment (mind-blast) [reply book/07-git-tools/1-git-tools.asc]: done change

File: book/07-git-tools/1-git-tools.asc (hunk 12 above)
Comment (mind-blast) [reply book/07-git-tools/1-git-tools.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 13 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 14 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 15 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 16 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 17 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 18 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 3 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done removing

Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 5 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 19 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 20 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 21 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change

File: book/02-git-basics/sections/recording-changes.asc (hunk 8 above)
Comment (mind-blast) [reply book/02-git-basics/sections/recording-changes.asc]: done change kaginhawahang to kaginhawaang
and also the sentence

File: book/07-git-tools/sections/advanced-merging.asc (hunk 22 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change para sa kontrol ng iyong source code and also pang-araw-araw na mga utos
done change maaaring

File: book/07-git-tools/sections/advanced-merging.asc (hunk 23 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

File: book/07-git-tools/sections/advanced-merging.asc (hunk 24 above)
Comment (mind-blast) [reply book/07-git-tools/sections/advanced-merging.asc]: done change

**Comment by mind-blast** [comment]
Done changing all suggested words. Thank you for proof reading.
